python/py-xraylarch/Portfile

```diff
@@ -27,16 +27,14 @@ long_description    Larch is a open-source library and set of applications for p
 homepage            https://xraypy.github.io/xraylarch
 
 # Find the latest at https://pypi.org/project/xraylarch/#files
-master_sites        https://files.pythonhosted.org/packages/ff/4a/cf7f494aaa3ea1ba15adf5806cef0ac5136affc7e8f7c2048999fa4e28d9/
+master_sites        https://files.pythonhosted.org/packages/00/52/6eb6628d913e7da91cbdc1f2aa4097b4b0559d72c2ad31914cb22804784d/
 distname            xraylarch-${version}-py3-none-any
 
-checksums           rmd160  0736ee2e7d6ba40c8c55dc7111cb3c3067454fd2 \
-                    sha256  92d558df4c8302a277ead4ad597920285abf3b7e69ca9ab13873e8dc319e452a \
-                    size    21056444
+checksums           rmd160  99f7bd396c449478da4ea8e199dfcb0d058d81f1 \
+                    sha256  a4331ec011b30ae028ef71dc9f03fb04efe8da4786467857b113ebc654e68019 \
+                    size    21152407
 
-python.versions     38 39 310
-python.pep517       yes
-python.pep517_backend
+python.versions     38 39 310 311 312
 
 if {${name} ne ${subport}} {
 
```

Comment:
there are tarballs on PyPI so please change to that instead of installing a `.whl` file.

Reply:
when doing that `distname` and `master_sites` become redundant, as well ass the `extract` stuff below, overriding of the `build` phase, and setting the `destroot.target`.

Reply:
Thank you for this. I was trying to get the tarball to work before the PR, and I had done everything you suggested _except_ removing the `destroot.target` .  When I also remove that override, then the tarball does build. Hooray.